libnymea/interfaces/rainfallsensor.json

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "sensor",
+    "description": "This interface represents a rainfall sensor and informs about how much millimeters rain has fallen in the given time period.",
+    "states": [
+        {
+            "name": "rainfallLastHour",
+            "type": "uint",
+            "unit": "MilliMeter",
+            "logged": true
+        },
+        {
+            "name": "rainfallLastDay",
+            "type": "uint",
+            "unit": "MilliMeter",
+            "logged": true,
+            "optional": true
```

Comment:
Hmm... So wouldn't it make the most sense if such an interface would inform whether it is raining right now or not?

Also, why last hour and last day? Why not last week/month/year?

Seems quite arbitrary and not really useful right now...

Reply:
> Seems quite arbitrary and not really useful right now...

maybe for you...
not for people with a rain gauge